Kickstarter-iOS/Views/Controllers/MessageBannerViewController.swift

```diff
@@ -2,28 +2,22 @@ import Foundation
 import Library
 import Prelude
 
-internal protocol MessageBannerViewControllerDelegate: class {
-  var messageBannerViewControllerContainer: UIView { get }
-  func messageBannerViewControllerIsHidden(_ isHidden: Bool)
+protocol MessageBannerViewControllerPresenting {
```

Comment:
What if we added `messageBannerViewController` as a `var` in the protocol too to be sure it is always a property on view controllers that are presenting it?